packages/collector/test/tracing/logger/console/test.js

```diff
@@ -0,0 +1,153 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
+
+const expect = require('chai').expect;
+const supportedVersion = require('@instana/core').tracing.supportedVersion;
+const config = require('../../../../../core/test/config');
+const testUtils = require('../../../../../core/test/test_util');
+const {
+  verifyHttpRootEntry,
+  verifyHttpExit,
+  verifyExitSpan
+} = require('../../../../../core/test/test_util/common_verifications');
+const globalAgent = require('../../../globalAgent');
+const ProcessControls = require('../../../test_util/ProcessControls');
+
+const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
+
+mochaSuiteFn('tracing/logger/console', function () {
+  this.timeout(config.getTestTimeout());
+
+  globalAgent.setUpCleanUpHooks();
+  const agentControls = globalAgent.instance;
+
+  const controls = new ProcessControls({
+    dirname: __dirname,
+    useGlobalAgent: true
+  });
+  ProcessControls.setUpHooks(controls);
+
+  /**
+   * CASES/NOTES
+   *
+   * - external loggers such as bunyan or winston use process.stdout internally
+   * - we internally use a logger component (e.g. logger.warn)
+   *    - core uses console by default, but nobody is using core directly without config
```

Comment:
risk: look at serverless console logger again to ensure there is no risk
otherwise this risk should be fine

Reply:
Is not a risk because serverless is currently initialised before core. That means any serverless console is not captured. Added a test & a comment